db/seeds/fincap_data/what_works_seeds.rb

```diff
@@ -0,0 +1,162 @@
+english_site = Comfy::Cms::Site.find_or_create_by(
+  label: 'en',
+  identifier: 'money-advice-service-en',
+  hostname: ENV['HOSTNAME'] || 'localhost:3000',
```

Comment:
Question unrelated to this PR probably: I cannot find this `HOSTNAME` anywhere in the [.env-example](https://github.com/moneyadviceservice/cms/blob/master/.env-example) – I see it's present in other seed files, but is it actually is use?

Reply:
Good point, I believe it is set depending on the env:
```bash
[root@***-tst-**** ~]$ echo $HOSTNAME
***-tst-****.dev.mas.local
```